nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -13,21 +13,25 @@
 
 from abc import ABC
 from abc import abstractmethod
-from typing import List, Tuple, TypeVar, Optional
+from typing import List
+from typing import Optional
+from typing import Tuple
+from typing import TypeVar
 
 import numpy as np
+
+from nncf.common.graph import NNCFGraph
+from nncf.common.graph import NNCFNode
 from nncf.common.graph.transformations.commands import TargetPoint
-from nncf.common.graph.transformations.commands import TransformationCommand
 from nncf.common.graph.transformations.commands import TargetType
+from nncf.common.graph.transformations.commands import TransformationCommand
 from nncf.common.tensor import NNCFTensor
-from nncf.common.graph import NNCFGraph
-from nncf.common.graph import NNCFNode
-from nncf.common.tensor_statistics.collectors import TensorStatisticCollectorBase
 from nncf.common.tensor_statistics.collectors import ReductionShape
+from nncf.common.tensor_statistics.collectors import TensorStatisticCollectorBase
 from nncf.common.utils.registry import Registry
 
-
 TModel = TypeVar('TModel')
+TTensor = TypeVar('TTensor')
```

Comment:
Is it Union[np.ndarray, torch.tensor]?

Reply:
Do you suggest to add backend specific import in common code?